docs/designers-developers/developers/tutorials/javascript/readme.md

```diff
@@ -18,4 +17,4 @@ The block editor introduced in WordPress 5.0 is written in JavaScript, with the
 5. [JavaScript Versions and Building](/docs/designers-developers/developers/tutorials/javascript/versions-and-building.md)
 6. [Scope your code](/docs/designers-developers/developers/tutorials/javascript/scope-your-code.md)
 7. [JavaScript Build Step](/docs/designers-developers/developers/tutorials/javascript/js-build-setup.md)
```

Comment:
Shouldn't we also reference `ESNext Syntax` inside `JavaScript Build Step`?

Reply:
Great suggestion, I'll add it there also.